components/Header.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import styled from 'styled-components';
+import HeaderText from '../static/bb_name.png';
+import BannerBear from '../static/banner_bear_no_logo.png';
+
+const StyledHeader = styled('header')`
+  width: 100vw;
```

Comment:
It would be better to do it like this due to the problems with scrollbar

```suggestion
  width: 100%;
```